lib/FieldType/Mapper/FloatFormMapper.php

```diff
@@ -61,7 +83,42 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
     }
 
     /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $names = $fieldDefinition->getNames();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($names);
+
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        NumberType::class,
```

Comment:
> What do you think?

I think that's a good example of a very bad comment :)
Joke aside, maybe at some point there were some issues but according to http://caniuse.com/#feat=input-number and my personal test page http://vrac.pwet.fr/input.html type=number input seems to work well for both Integer and Float

See also https://github.com/symfony/symfony/issues/8106